cityofaustin/atd-moped
# Remove previous project components JS

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/11313

This PR: 
- removes code for the previous components map
- removes some `@turf` tools that we no longer use
- updates some outdated comments about the `ComponentsDrawControl` component
- patches a failing test

## Testing
**URL to test:** 

https://deploy-preview-962--atd-moped-main.netlify.app/moped/ or local

**Steps to test:**
1. Create a new signal project - you should see the project summary map load with the signal that you chose when saving finishes
2. Go to the **Map** tab
3. The map should center on the signal that you chose
4. Test out adding new components:
   - Add a line component, select some lines, and draw some too
   - Add a point component, select some lines, and draw some too
   - Add a Signal - PHB component
5. Test out editing those components
6. Click on the list items of component to make sure they still highlight and the others grey out
7. Delete a component
8. Zoom to components using the magnifier icon
9. Change the base map to satellite and back to street view
10. Test the geocoder by searching for **Austin City Hall** or something else
11. Delete all components and then go to the project summary view. You should see the fallback component that says **Define and map the assets and features included in this project** with the **Add Components** button below it
12. Click that button and the components map should open

---
#### Ship list
